pkg/machinery/resources/secrets/kubernetes_root.go

```diff
@@ -30,12 +30,12 @@ type KubernetesRoot = typed.Resource[KubernetesRootSpec, KubernetesRootRD]
 //
 //gotagsrewrite:gen
 type KubernetesRootSpec struct {
-	Name          string   `yaml:"name" protobuf:"1"`
-	Endpoint      *url.URL `yaml:"endpoint" protobuf:"2"`
-	LocalEndpoint *url.URL `yaml:"local_endpoint" protobuf:"3"`
-	CertSANs      []string `yaml:"certSANs" protobuf:"4"`
-	APIServerIPs  []net.IP `yaml:"apiServerIPs" protobuf:"5"`
-	DNSDomain     string   `yaml:"dnsDomain" protobuf:"6"`
+	Name          string       `yaml:"name" protobuf:"1"`
+	Endpoint      *url.URL     `yaml:"endpoint" protobuf:"2"`
+	LocalEndpoint *url.URL     `yaml:"local_endpoint" protobuf:"3"`
+	CertSANs      []string     `yaml:"certSANs" protobuf:"4"`
+	APIServerIPs  []netip.Addr `yaml:"apiServerIPs" protobuf:"14"`
```

Comment:
this seems like a problem in the previous iterations of `protobuf` resources - there should be `net.IP` type ever. I updated the type and change proto tag to avoid a failure to unmarshal